sprint0/sprint0/Weapons/Arrow.cs

```diff
@@ -66,6 +69,14 @@ public void Update()
             if (IsAlive())
             {
                 rotation = defaultAngle - dir.ToRadians();
+                if (dir == Direction.e || dir == Direction.w)
+                {
+                    Location = new Rectangle((int)Location.X, (int)Location.Y, (int)(height * Game1.Scale), (int)(width * Game1.Scale));
+                }
+                else
+                {
+                    Location = new Rectangle((int)Location.X, (int)Location.Y, (int)(width * Game1.Scale), (int)(height * Game1.Scale));
```

Comment:
this doesn't seem different either

Reply:
width and height get flipped when the arrow is facing east or west. The entire issue that the arrow when facing east or west needs to get drawn with the rectangle of (width, height) but the collision box needs to detect with rectangle of (height, width)

Reply:
ok makes sense. can we move this to the constructor since I think we only need to set width/height once depending on the direction? I can just approve for now if you want to do that later